.github/workflows/add-issues-and-prs-to-project.yml

```
@@ -0,0 +1,19 @@
+name: Add issues and pull requests to project
+
+on:
+  issues:
+    types:
+      - opened
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@main
+        with:
+          project-url: https://github.com/orgs/iTwin/projects/3/views/1
```

I think there's no need for views here
```suggestion
          project-url: https://github.com/orgs/iTwin/projects/3
```

---

good catch, changed